### PR TITLE
Adding compat data for Blob

### LINKED ISSUE #969: Add compat data for Blob
MDN pages to migrate:

- [ ] [Web/API/Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob)
- [ ] [Web/API/Blob/Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob/Blob)
- [ ] [Web/API/Blob/size](https://developer.mozilla.org/en-US/docs/Web/API/Blob/size)
- [ ] [Web/API/Blob/type](https://developer.mozilla.org/en-US/docs/Web/API/Blob/type)
- [ ] [Web/API/Blob/slice](https://developer.mozilla.org/en-US/docs/Web/API/Blob/slice)

The data should be migrated into the file `api/Blob.json`.

For more information on how to help with this work, see https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables

### PR DESCRIPTION
**Pull request by CalmBit**
Hey there all! Just dropping this here 😄 

Quick question about the notes in the first `Blob` entry though - should those be retained? That information is essentially wholesale duplicated in the actual compat data for `slice`, so it seems a little redundant to restate.

Fixes #969

### CODE REVIEW

File: api/Blob.json [hunk 1]
```diff
@@ -0,0 +1,271 @@
...
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": "5.1",
+              "prefix": "webkit",
+              "notes": "This was implemented in <a href='http://trac.webkit.org/changeset/83873'>534.29</a>."
```
Comment (Elchi3) [inline api/Blob.json]: Remove this note. too. The bug that implemented something is not relevant to anyone anymore.

File: api/Blob.json [hunk 2]
```diff
@@ -0,0 +1,271 @@
...
+                "prefix": "webkit",
+                "notes": "The <code>slice()</code> method had initially taken <code>length</code> as the second argument to indicate the number of bytes to include in the new <code>Blob</code>. If you specified values such that <code>start + length</code> exceeded the size of the source <code>Blob</code>, the returned <code>Blob</code> contained data from the start index to the end of the source <code>Blob</code>.&nbsp;That version of the <code>slice()</code> was implemented in <a href='https://hg.mozilla.org/mozilla-central/rev/1b3947ed93c6'>Firefox 4</a>, <a href='http://trac.webkit.org/changeset/55670'>WebKit</a>, and <a href='http://www.opera.com/docs/specs/presto28/file/#blob'>Opera 11.10</a>. However, since that syntax is differed from <a href='/en-US/JavaScript/Reference/Global_Objects/Array/slice'><code>Array.slice()</code></a> and <a href='/en-US/JavaScript/Reference/Global_Objects/String/slice'><code>String.slice()</code></a>, Gecko and WebKit removed support and added support for the new syntax as <code>Blob.slice()</code>/<a href='http://trac.webkit.org/changeset/83873'><code>Blob.webkitSlice</code></a>."
```
Comment (Elchi3) [inline api/Blob.json]: Let's remove these lengthy notes here, too. They are way too long and we have the situation explained in the wiki at the top of the page already. The important (machine-readable) info here are the prefixes.

**Comment by CalmBit** [comment]
Alrighty, removed some of those redundant notes - there's a couple more that are a bit long, but they seem to be relevant to actual implementation-specific quirks, so I kept them in. Sorry about that!

**Comment by CalmBit** [comment]
Done! :smile:

**Pull request merged**